api/templates/api.mustache

```diff
@@ -414,18 +414,18 @@ func (a *{{{classname}}}Service) {{nickname}}Execute(r {{#structPrefix}}{{&class
 	body, err := GunzipIfNeeded(localVarHTTPResponse)
 	if err != nil {
 		body.Close()
-		return {{#returnType}}localVarReturnValue, {{/returnType}}_fmt.Errorf("%s%w", errorPrefix, err)
+		return {{#returnType}}localVarReturnValue, {{/returnType}}_fmt.Errorf("%s: %w", errorPrefix, err)
 	}
 	localVarBody, err := _ioutil.ReadAll(body)
 	body.Close()
 	if err != nil {
-		return {{#returnType}}localVarReturnValue, {{/returnType}}_fmt.Errorf("%s%w", errorPrefix, err)
+		return {{#returnType}}localVarReturnValue, {{/returnType}}_fmt.Errorf("%s: %w", errorPrefix, err)
 	}
 	err = a.client.decode(&localVarReturnValue, localVarBody, localVarHTTPResponse.Header.Get("Content-Type"))
 	if err != nil {
 		newErr := GenericOpenAPIError{
 			body:  localVarBody,
-			error: _fmt.Sprintf("%s%s", errorPrefix, err.Error()),
+			error: _fmt.Sprintf("%s: %s", errorPrefix, err.Error()),
```

Comment:
Hmmm why not this?
```suggestion
			error: _fmt.Sprintf("%s: %w", errorPrefix, err),
```

Reply:
I hadn't tried this...while it does compile, it gives a linting warning everywhere: `fmt.Sprintf call has error-wrapping directive %w, which is only supported for functions backed by fmt.Errorf`, so it doesn't seem ideal.

Maybe there's a way to do this with error wrapping and setting the `model` field for the `GenericOpenAPIError` ... I'll look into it.

Reply:
Oh I completely missed that this is `Sprintf`, not `Errorf`. In that case `%s` is fine 👍

Reply:
I changed the above one to `%s` as well so that all of them are consistent now. Not sure why it was a `%v`, must have just been a hold-over from before. Basically this entire chain of logic is just building up a string.

On that note, I looked at trying to do something more idiomatic like building up the error with `%w` and `fmt.Errorf` and didn't have much success. Untangling that from the error types in the template file here didn't seem to work very well and you'd end up converting back and forth between `error` and `string` all the time anyway. 

It would be nice to revisit this with https://github.com/influxdata/influx-cli/issues/142. I'm liking the idea more and more of a pre-flight request for server-specific commands and erroring out before even trying to run the command, and then just reverting all of this error building stuff back to the defaults from the template.